tensorflow/python/ops/math_ops.py

```diff
@@ -2170,13 +2174,10 @@ def add_n(inputs, name=None):
 def accumulate_n(inputs, shape=None, tensor_dtype=None, name=None):
   """Returns the element-wise sum of a list of tensors.
 
-  Optionally, pass `shape` and `tensor_dtype` for shape and type checking,
-  otherwise, these are inferred.
-
```

Comment:
Maybe we can keep this?

Reply:
Added this back in, along with an additional note that `shape` will override incomplete shape information on the output of the op, which can affect type inference for downstream ops.